### PR TITLE
Use tox instead of manual script

### PR DESCRIPTION
**Pull request by sileht**

### CODE REVIEW

**Comment by sileht** [comment]
Pull-request updated, HEAD is now 9741454b368526d68be6e91a2a5958661805a4ee

**Comment by sileht** [comment]
Pull-request updated, HEAD is now bce95832ccdb70c6157efe83316ae771639f42d4

**Comment by sileht** [comment]
Pull-request updated, HEAD is now cc3dcb88c36967a5a7cb23cc09c47ec1034ca682

File: tox.ini [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+deps =
+  gnocchi[postgresql,file]
+  pifpaf[gnocchi]
+whitelist_externals = bash
+commands =
+  {toxinidir}/run-tests.sh
+
+[testenv:pep8]
+deps = hacking>=0.12,<0.13
+       bashate
```
Comment (jd) [inline tox.ini]: does not sound used.

File: tox.ini [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+sitepackages = False
+deps =
+  gnocchi[postgresql,file]
+  pifpaf[gnocchi]
+whitelist_externals = bash
+commands =
+  {toxinidir}/run-tests.sh
+
+[testenv:pep8]
+deps = hacking>=0.12,<0.13
```
Comment (jd) [inline tox.ini]: can we not use this non maintained stuff and use flake8 plugins?

**Comment by sileht** [comment]
Pull-request updated, HEAD is now 01caf789755c2fbe74a0b599e11edf6df45f8b2d

**Comment by sileht** [comment]
Pull-request updated, HEAD is now 682e9fdf3372d49d23c968c961f556640e4ea5d0

**Comment by sileht** [comment]
Pull-request updated, HEAD is now f3fae302f4589dd95c7cab844f67cffec0de9f5b

**Comment by sileht** [comment]
Pull-request updated, HEAD is now 0ade1753fea9103a0d18e44968ef9352d4a0bf42

**Comment by sileht** [comment]
Pull-request updated, HEAD is now 532e05bd633d765909cc038a2dba0120648f0bc7

**Pull request merged**